basket/settings.py

```diff
@@ -244,10 +244,11 @@ def path(*args):
 KOMBU_FERNET_KEY_PREVIOUS = config('KOMBU_FERNET_KEY_PREVIOUS', None)
 CELERY_TASK_ALWAYS_EAGER = config('CELERY_TASK_ALWAYS_EAGER', DEBUG, cast=bool)
 CELERY_TASK_SERIALIZER = 'json'
-CELERY_TASK_ACKS_LATE = config('CELERY_TASK_ACKS_LATE', False, cast=bool)
+CELERY_TASK_ACKS_LATE = config('CELERY_TASK_ACKS_LATE', True, cast=bool)
```

Comment:
According to Celery docs, this means that the task may be multiple times executed. I believe this is alright from a data standpoint but maybe it reduces our SF api quota faster?

Reply:
That's true. In theory this should only apply if the task doesn't complete before the worker is killed. So this should ensure that the tasks are executed fully at least once even if the pod or node is suddenly destroyed. In theory it should not mean much duplication. I'll definitely be keeping an eye on it thought. Good call-out.